src/hyp3_autorift/process.py

```diff
@@ -319,6 +331,45 @@ def apply_landsat_filtering(reference_path: str, secondary_path: str) \
     return reference_path, reference_zero_path, secondary_path, secondary_zero_path
 
 
+def get_lat_lon_from_ncfile(ncfile: Path) -> Tuple[float, float]:
+    with Dataset(ncfile) as ds:
+        var = ds.variables['img_pair_info']
+        return var.latitude, var.longitude
```

Comment:
```suggestion
def get_lat_lon_from_ncfile(ncfile: Path) -> Tuple[float, float]:
    with Dataset(ncfile) as ds:
        var = ds.variables['img_pair_info']
    return var.latitude, var.longitude
```

Reply:
If we can still access `var` outside of the open dataset, I think this is a better syntax.